server/core/src/https/v1.rs

```diff
@@ -1425,9 +1425,18 @@ pub async fn auth_valid(
     to_axum_response(res)
 }
 
+#[cfg(debug_assertions)]
+pub async fn debug_ipinfo(
+    State(state): State<ServerState>,
+    TrustedClientIp(ip_addr): TrustedClientIp,
+) -> impl IntoResponse {
+    let res = state.qe_r_ref.handle_debug_ipinfo(ip_addr);
```

Comment:
Could just skip the qe_r_ref here since this doesn't do anything to the query server, it's axum-level only. Makes the code simpler too.

Reply:
wopsies :)